zulipterminal/config/keys.py

```diff
@@ -200,6 +200,11 @@
         'help_text': 'View stream description',
         'key_category': 'stream_list',
     }),
+    ('USER_INFO', {
+        'keys': {'i'},
```

Comment:
You could extract the README and keys into a sepearte commit as well. As it stands, this commit has a lot of file changes.